language/src/regent/ast.t

```diff
@@ -666,4 +666,11 @@ function ast.mapreduce_stat_postorder(map_fn, reduce_fn, node, init)
     reduce_fn, node, init, is_stat_node)
 end
 
+function ast.get_base_indexed_node(node, previous_index)
```

Comment:
This seems like the sort of thing that would be more appropriate in `ast_util.t`?

Reply:
Right, thanks.